src/stackset.ts

```diff
@@ -645,11 +645,7 @@ export class StackSet extends Resource implements IStackSet {
       managedExecution: {
         Active: props.managedExecution ?? true,
       },
-      operationPreferences: undefinedIfNoKeys({
-        regionConcurrencyType: props.operationPreferences?.regionConcurrencyType,
-        maxConcurrentPercentage: props.operationPreferences?.maxConcurrentPercentage,
-        failureTolerancePercentage: props.operationPreferences?.failureTolerancePercentage,
-      }),
+      operationPreferences: props.operationPreferences,
```

Comment:
```suggestion
      operationPreferences: undefinedIfNoKeys(props.operationPreferences),
```

Reply:
it seems stupid now but if someone previously had `operationPreferences = {}` they would see a change in the new code (i.e. previously would be undefined and now would be {})